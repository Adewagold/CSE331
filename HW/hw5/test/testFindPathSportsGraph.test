# Test BFS on sports graph

# build the graph
LoadGraph sports sports.tsv

# test BFS order
# find the shortest path Howard to Michael
FindPath sports Howard Michael

# find the shortest path Michael to Howard
FindPath sports Michael Howard