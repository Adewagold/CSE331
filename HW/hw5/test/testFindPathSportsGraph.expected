# Test BFS on sports graph

# build the graph
loaded graph sports

# test BFS order
# find the shortest path Howard to Michael
path from Howard to Michael:
Howard to Jeff via NFL
Jeff to Michael via MLB

# find the shortest path Michael to Howard
path from Michael to Howard:
Michael to Jeff via MLB
Jeff to Howard via NFL